bundles/org.openhab.binding.solarwatt/README.md

```diff
@@ -0,0 +1,242 @@
+# Solarwatt Binding
+
+Binding to query a [solarwatt](https://www.solarwatt.de/) [energy manager](https://www.solarwatt.de/energie-management/energymanager) and read the values of all attached devices.
+
+All supported values and devices were discovered while playing with my own energy manager.
+
+## Supported Things
+
+| Thing Type ID | Devices |
+|------|---------------|
+| energymanager | EnergyManager itself. |
+| location | Location part of the EnergyManager. | 
+| pvplant | Power producing part of the EnergyManager. |
+| gridflow | Grid interaction part of the EnergyManager. |
+| inverter | inverter producing AC current; e.g. MyReserve, Fronius |
+| batteryconverter | battery storage systems; e.g. MyReserve |
+| powermeter | powermeters; e.g. S0BusCounter, MyReserve |
+| evstation | electric-vehicle charging station; e.g. Keba Wallbox |
+
+## Discovery
+
+You have to enter the hostname or ip-address of the energymanager itself.
+The attached devices and supported channels are discovered automatically.
+
+## Thing Configuration
+
+| Property | Default | Required | Description |
+|----------|---------|----------|-------------|
+| hostname | None | Yes | hostname or ip-address of the energy manager. |
```

Comment:
Can you mention to which Thing Type UID this parameter applys to? It seems that it applys only to the energymanager. There seems also a refresh and a rescan parameter.

What about the other Thing Type UIDs? They need at least one configuration parameter to identify them.